### ISSUE TITLE
[Bug]: Not connected can't continue!

### ISSUE THREAD

**Issue by crag364** [homebridge-eufy-security/plugin#288]
### What happened?

Hi,

I keep getting this message on my log. I have followed all the troubleshooting instructions, and cleared the app data and re-logged in mutiple times. Just cannot get past this message. Even tried updating to the beta release. Any advice?

Thanks,

Craig

### Device Type

Doorbell (Supported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[05/02/2023, 18:13:57] Homebridge v1.6.0 (HAP v0.11.0) (Homebridge 4D6A) is running on port 51761.
[05/02/2023, 18:13:58] [homebridge-eufy-security] Launched child bridge with PID 7486
[05/02/2023, 18:14:00] Registering platform 'homebridge-eufy-security.EufySecurity'
[05/02/2023, 18:14:02] [homebridge-eufy-security] Loaded homebridge-eufy-security v2.2.0-beta.17 child bridge successfully
[2/5/2023, 6:14:02 PM] [EufySecurity-2.2.0-beta.17] WARN:  warning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1
[2/5/2023, 6:14:02 PM] [EufySecurity-2.2.0-beta.17] INFO:  Country set: GB
[2/5/2023, 6:14:02 PM] [EufySecurity-2.2.0-beta.17] INFO:  Finished initializing!
[05/02/2023, 18:14:03] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-eufy-security) is running on port 58785.
[2/5/2023, 6:14:07 PM] [EufySecurity-2.2.0-beta.17] WARN:  There was a Captcha request while login in. This cannot be fulfilled by the plugin itself. Please login using the configuration wizard (settings) in the Homebridge UI plugins tab.
[2/5/2023, 6:14:07 PM] [EufySecurity-2.2.0-beta.17] ERROR: Not connected can't continue!
```

### config

```shell
{
            "platform": "EufySecurity",
            "ignoreStations": [],
            "ignoreDevices": [],
            "cleanCache": true,
            "username": "*",
            "password": "*",
            "country": "GB",
            "deviceName": "Homebridge",
            "_bridge": {
                "username": "0E:81:12:58:D9:37",
                "port": 58785
            },
            "enableDetailedLogging": true
        },
```

**Comment by metalcated** [comment]
I am getting this as well. I asked in Discord but crickets :)

**Comment by metalcated** [comment]
Possibly something to do with the new "Web Portal Access" Safety Pin?

**Comment by metalcated** [comment]
Okay so I validated that I can see my "device" in Trusted on the app and its added correctly. Still after I see this in logs:

```
[2/7/2023, 5:13:17 PM] [EufySecurity-2.2.0-beta.17] WARN:  There was a Captcha request while login in. This cannot be fulfilled by the plugin itself. Please login using the configuration wizard (settings) in the Homebridge UI plugins tab.
[2/7/2023, 5:13:17 PM] [EufySecurity-2.2.0-beta.17] DEBUG: EufyClient connected false
[2/7/2023, 5:13:17 PM] [EufySecurity-2.2.0-beta.17] ERROR: Not connected can't continue!
```

**Comment by theST4GZ** [comment]
Im experiencing the same thing, I've attached my logs if they help
[configui-server.log](https://github.com/homebridge-eufy-security/plugin/files/10681637/configui-server.log)
[eufy-log.log](https://github.com/homebridge-eufy-security/plugin/files/10681638/eufy-log.log)
[log-lib.log](https://github.com/homebridge-eufy-security/plugin/files/10681639/log-lib.log)

**Comment by metalcated** [comment]
Any chance this has been reviewed and someone is at least taking a look at this? Anything additional we can do to assist/help with? More than happy to do so. :) Thanks!

**Comment by Z1mDMan** [comment]
My doorbell is not usable with this bug. It was fixed in beta 17 but it seems to be back.

**Comment by schliemann** [comment]
I'm experiencing the same. Will try and update the eufy security client on my Pi and see if it fixes the login. If this fixes it there are two options for people:

1. Update package themselves if they can't wait on new release.
2. Await new release of beta and non-beta release - might not be before next week.

**Comment by schliemann** [comment]
I can confirm that eufy security client 2.4.2 works.

![image](https://user-images.githubusercontent.com/10818044/221025555-f7ec6426-e174-4f21-a64d-42928599e10f.png)

**Comment by schliemann** [comment]
Will try to update the beta tomorrow.

**Comment by metalcated** [comment]
Do you have a donation link? Much appreciated on the work getting this back to functional. Thanks bro!

**Comment by schliemann** [comment]
Beta should be fixed now. I'm having some trouble getting master published. Need some help from @thieren or @lenoxys .

**Comment by nukithelegend** [comment]
I created a guest admin account instead of my main account and that worked for me

**Comment by c0mm0n** [comment]
Having this issue since today, everything was fine for weeks, using latest beta.
I tried guest account / admin account / family sharing account without success.
Any workaround ?

**Comment by Haglerd** [comment]
> Having this issue since today, everything was fine for weeks, using latest beta. I tried guest account / admin account / family sharing account without success. Any workaround ?

Same thing is happening for me. Exact same issue from this thread. I'm on the latest Beta as of today. I've looked at the common issues and relogged in  to no avail

[4/14/2023, 4:41:01 PM] [EufySecurity-2.2.0-beta.19] WARN:  There was a Captcha request while login in. This cannot be fulfilled by the plugin itself. Please login using the configuration wizard (settings) in the Homebridge UI plugins tab.
[4/14/2023, 4:41:01 PM] [EufySecurity-2.2.0-beta.19] ERROR: Not connected can't continue!

**Comment by c0mm0n** [comment]
@Haglerd uninstalling the whole plugin, deleting its folder and restarting from scratch solved my issue. 
No idea if there is a less destructive alternative.

**Comment by langluc** [comment]
I have the same problem. 



{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":6640,"level":30,"msg":"Finished shutdown!","time":"2023-08-22T06:49:00.738Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7610,"level":40,"msg":"warning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1","time":"2023-08-22T06:49:39.902Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7610,"level":30,"msg":"Country set: IT","time":"2023-08-22T06:49:39.931Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7610,"level":30,"msg":"Finished initializing!","time":"2023-08-22T06:49:39.933Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7610,"level":50,"msg":"Not connected can't continue!","time":"2023-08-22T06:49:42.923Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7610,"level":30,"msg":"Finished shutdown!","time":"2023-08-22T07:02:04.926Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7922,"level":40,"msg":"warning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1","time":"2023-08-22T07:02:43.317Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7922,"level":30,"msg":"Country set: IT","time":"2023-08-22T07:02:43.347Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7922,"level":30,"msg":"Finished initializing!","time":"2023-08-22T07:02:43.348Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7922,"level":50,"msg":"Not connected can't continue!","time":"2023-08-22T07:02:47.501Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":7922,"level":30,"msg":"Finished shutdown!","time":"2023-08-22T16:30:06.380Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":8731,"level":40,"msg":"warning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1","time":"2023-08-22T16:30:44.493Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":8731,"level":30,"msg":"Country set: IT","time":"2023-08-22T16:30:44.523Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":8731,"level":30,"msg":"Finished initializing!","time":"2023-08-22T16:30:44.525Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":8731,"level":50,"msg":"Not connected can't continue!","time":"2023-08-22T16:30:56.584Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":8731,"level":30,"msg":"Finished shutdown!","time":"2023-08-22T16:32:57.908Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":9331,"level":40,"msg":"warning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1","time":"2023-08-22T16:45:31.746Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":9331,"level":30,"msg":"Country set: IT","time":"2023-08-22T16:45:31.774Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":9331,"level":30,"msg":"Finished initializing!","time":"2023-08-22T16:45:31.776Z","v":0}
{"name":"[EufySecurity-2.2.2]","hostname":"homebridge","pid":9331,"level":50,"msg":"Not connected can't continue!","time":"2023-08-22T16:45:43.876Z","v":0}

**Comment by lenoxys** [comment]
should be fixed try 2.2.14
@baraoresmungao @langluc @Haglerd

**Comment by langluc** [comment]
I’ll try but with no success. Maybe the problem is the hardware of raspberry pi2. Luca ***@***.*** giorno 8 set 2023, alle ore 21:53, lenoxys ***@***.***> ha scritto:﻿
should be fixed try 2.2.14
@baraoresmungao @langluc @Haglerd

**Comment by lenoxys** [comment]
Do you see an error showing up ? Can you send logs ? Make sure to it refresh on plugin settings, RENATER captcha and 2fa and then reboot

**Comment by langluc** [comment]
This is log file Luca ***@***.*** giorno 9 set 2023, alle ore 00:05, lenoxys ***@***.***> ha scritto:﻿
Do you see an error showing up ? Can you send logs ? Make sure to it refresh on plugin settings, RENATER captcha and 2fa and then reboot

**Comment by lenoxys** [comment]
Attachment doesn't work when sends thru email

**Comment by langluc** [comment]
[Unknown 3.zip](https://github.com/homebridge-eufy-security/plugin/files/12565180/Unknown.3.zip)

**Comment by lenoxys** [comment]
It's weird I miss the plugin logs looks like you don't restart the home bridge : 

Can you try this steps : 

1. Disable the Eufy Security plugin
2. Reboot Homebridge
3. Click on refresh button within Eufy Security Plugin settings
4. You should be prompted for Captcha and 2FA
5. Then devices should appear.
6. Click save.
7. Enable the plugin again
8. Reboot Homedridge

and then provide the logs file.

**Comment by langluc** [comment]
I see the new version and now, after upgrade, it works. Thanks you Luca ***@***.*** giorno 10 set 2023, alle ore 13:00, lenoxys ***@***.***> ha scritto:﻿
It's weird I miss the plugin logs looks like you don't restart the home bridge :
Can you try this steps :

Disable the Eufy Security plugin
Reboot Homebridge
Click on refresh button within Eufy Security Plugin settings
You should be prompted for Captcha and 2FA
Then devices should appear.
Click save.
Enable the plugin again
Reboot Homedridge

and then provide the logs file.

**Comment by lenoxys** [comment]
ha cool thanks for your feedback, I'm closing this. If you have futur issue don't hesitate to look at https://github.com/homebridge-eufy-security/plugin/wiki/Common-Issues

and feel free to open another issue.
